utils/test/README.md

```diff
@@ -0,0 +1,3 @@
+# Test Utils
+
+This directory contains all files that provide utility to all tests. See [Test utilities](../../docs/Test.md#test-utilities) .
```

Comment:
See https://github.com/matthew-chirgwin/strimzi-ui/tree/master/utils/build for example of how this file could look